source/nvda.pyw

```diff
@@ -251,7 +256,9 @@ if customVenvDetected:
 	log.warning("NVDA launched using a custom Python virtual environment.")
 if globalVars.appArgs.changeScreenReaderFlag:
 	winUser.setSystemScreenReaderFlag(True)
-#Accept wm_quit from other processes, even if running with higher privilages
+# Accept wm_quit from other processes, even if running with higher privilages
+# This allows for downgrading between NVDA versions, deprecated in 2022.1
+# in 2022.1 remove the call to ChangeWindowMessageFilter (#12286)
 if not ctypes.windll.user32.ChangeWindowMessageFilter(winUser.WM_QUIT,1):

```

Comment:
Can you make `1` a constant or enum: 
MSGFLT_ADD=1 # Adds the message to the filter. This has the effect of allowing the message to be received.
MSGFLT_REMOVE=2 # Removes the message from the filter. This has the effect of blocking the message.

Reply:
added these to `winUser.py`